### PR TITLE
Place common docker compose files to integration docker container

### PR DESCRIPTION
**Pull request by qoega**
I hereby agree to the terms of the CLA available at: https://yandex.ru/legal/cla/?lang=en

Changelog category (leave one):
- Build/Testing/Packaging Improvement

Changelog entry (a user-readable short description of the changes that goes to CHANGELOG.md):

Place common docker compose files to integration docker container.

Detailed description / Documentation draft:

Less dependencies on repository to run test.

By adding documentation, you'll allow users to try your new feature immediately, not when someone else will have time to document it later. Documentation is necessary for all features that affect user experience in any way. You can add brief documentation draft above, or add documentation right into your patch as Markdown files in [docs](https://github.com/ClickHouse/ClickHouse/tree/master/docs) folder.

If you are doing this for the first time, it's recommended to read the lightweight [Contributing to ClickHouse Documentation](https://github.com/ClickHouse/ClickHouse/tree/master/docs/README.md) guide first.

### CODE REVIEW

File: tests/integration/helpers/cluster.py [hunk 1]
```diff
@@ -67,6 +67,17 @@ def get_odbc_bridge_path():
             return '/usr/bin/clickhouse-odbc-bridge'
     return path
 
+def get_docker_compose_path():
+    compose_path = os.environ.get('DOCKER_COMPOSE_DIR')
+    if compose_path is not None:
+        return os.path.dirname(compose_path)
+    else:
+        if os.path.exists(os.path.dirname('/compose/')):
```
Comment (alesapin) [inline tests/integration/helpers/cluster.py:75]: Why we need `dirname` here?

File: tests/integration/helpers/cluster.py [hunk 2]
```diff
@@ -67,6 +67,17 @@ def get_odbc_bridge_path():
...
+            return os.path.dirname('/compose/') #default in docker runner container
+        else:
+            print("Fallback docker_compose_path to LOCAL_DOCKER_COMPOSE_DIR: {}".format(LOCAL_DOCKER_COMPOSE_DIR))
```
Comment (alesapin) [inline tests/integration/helpers/cluster.py:78]: Almost everybody who runs it locally will see this message and it looks like you are making something wrong. Maybe something like `"Will use compose files from {}"`?

**Review by alesapin — APPROVED** [review]
Ok, but maybe slightly improve message (can merge without tests)

File: tests/integration/helpers/cluster.py (hunk 2 above)
Comment (qoega) [reply tests/integration/helpers/cluster.py:78]: Do you run integration tests without runner and runner docker container?
It is used ONLY when there is no `yandex/clickhouse-integration-tests-runner`. It means you run pytest manually without runner script.
And `yes` - it is a warning as it is not the same way CI runs it and it is better to see which YaMLs are used instead.

File: tests/integration/helpers/cluster.py (hunk 1 above)
Comment (qoega) [reply tests/integration/helpers/cluster.py:75]: No real purpose.

File: tests/integration/helpers/cluster.py (hunk 2 above)
Comment (alesapin) [reply tests/integration/helpers/cluster.py:78]: `> Do you run integration tests without runner and runner docker container?`
To be honest, I always do)

**Comment by alesapin** [comment]
Ok, let's merge

**Pull request merged**